### PR TITLE
Fix ci cuda problems

### PR DESCRIPTION
**Pull request by larroy**
## Description ##

update CUDNN from 7.5.1.10 to 7.6.0.64 in ubuntu_gpu_cu101

### CODE REVIEW

**Comment by marcoabreu** [comment]
Guys I feel like we have half a dozen PRs for the same thing now. Who owns this? Can we please consolidate these efforts? Everyone states that they fixed something, but nothing is merged or seems to be tested. What's going on here?

**Comment by larroy** [comment]
We are trying to fix it. Opening PRs to see results and compare. Not sure what's your point. We are also using the dev CI for that matter, but it's a complex interaction of the environment and the software inside the nvidia docker container.

File: tests/python/unittest/test_model_parallel.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 import numpy as np
 import mxnet as mx
+from mxnet.test_utils import *
```
Comment (apeforest) [inline tests/python/unittest/test_model_parallel.py]: What is this change for?

File: src/operator/cross_device_copy.cc [hunk 2]
```diff
@@ -60,6 +60,16 @@ class CrossDeviceCopyProp : public OperatorProperty {
     return true;
   }
 
+  bool InferType(std::vector<int> *in_type,
```
Comment (apeforest) [inline src/operator/cross_device_copy.cc]: How would this fix CI?

File: tests/python/unittest/test_model_parallel.py (hunk 1 above)
Comment (larroy) [reply tests/python/unittest/test_model_parallel.py]: This is not needed, thanks.

File: src/operator/cross_device_copy.cc (hunk 2 above)
Comment (larroy) [reply src/operator/cross_device_copy.cc]: Looks like I pulled an additional commit while rebasing, seems obvious.

**Pull request closed without merging**